app/scripts/modules/alicloud/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "buildOnSave": false,
+  "compileOnSave": true,
+  "compilerOptions": {
+    "allowJs": false,
+    "baseUrl": "./src",
+    "declaration": true,
+    "declarationDir": "lib",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "jsx": "react",
+    "lib": ["es2016", "dom", "es2017.object"],
+    "moduleResolution": "node",
+    "module": "esnext",
+    "noEmitHelpers": false,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": false, // should really get to a place where we can turn this on
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "outDir": "lib",
+    "pretty": true,
+    "removeComments": false,
+    "rootDir": "./src",
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "inlineSources": true,
+    "strictNullChecks": false, // should really get to a place where we can turn this on
+    "target": "es6",
+    "typeRoots": ["../../../../node_modules/@types"],
+    "paths": {
```

Comment:
```suggestion
    "paths": {
     "@spinnaker/core": ["../../core/lib"],
      "core/*": ["../../core/lib/*"],
```

Reply:
I think the build is failing because it can't resolve the `core` imports; this should do the trick!
Additionally, you'll want to add alicloud to the list of `paths` in the root `tsconfig.json`.

Reply:
HI @maggieneterval Thanks a lot. According to your suggestion, the CI issue is fixed. Please review the pr again and let me know if there is any other problems.

Reply:
Glad that fixed the CI issue! Sorry if you already did this and I'm just missing it, but could you please add alicloud to the list of `paths` in the root `tsconfig.json` [here](https://github.com/spinnaker/deck/blob/master/tsconfig.json#L20)?

Reply:
Done

Reply:
Great, thanks! The rest looks fine to me, deferring to @christopherthielen and @Jammy-Louie for the final thumbs-up as they have actually added new modules to Deck and I have not :smile:

Reply:
Thanks!

Reply:
Looks like you've updated the correct files to add the module to deck correctly to me